packages/schemix/src/schemix/__init__.py

```
@@ -1,2 +1,35 @@
+    "AsyncConnection",
+    "SQLiteConnection",
+    "PostgreSQLConnection",
+    "Dialect",
+    # Schema utilities
+    "generate_create_table_sql",
+    "generate_column_sql",
+    "generate_foreign_key_constraint_sql",
+]
+
+
 def hello() -> str:
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)